examples/convert_empty_room.py

```diff
@@ -0,0 +1,91 @@
+"""
+.. _ex-convert-mne-sample:
+
+======================================
+Storing empty room data in BIDS format
+======================================
+
+This example demonstrates how to store empty room data in BIDS format
+and how to retrieve them.
+"""
+
+# Authors: Mainak Jas <mainakjas@gmail.com>
+#
+# License: BSD (3-clause)
+
+###############################################################################
+# Let us first import mne_bids
+
+import os.path as op
+
+from datetime import datetime
+
+import mne
+from mne.datasets import sample
+
+from mne_bids import write_raw_bids, read_raw_bids, make_bids_basename
+
+###############################################################################
+# And define the paths and event_id dictionary.
+
+data_path = sample.data_path()
+output_path = op.join(data_path, '..', 'MNE-sample-data-bids')
+raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
```

Comment:
```Python
# paths to our raw data
data_path = sample.data_path()
raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')

# place to but converted data
output_path = op.join(data_path, '..', 'MNE-sample-data-bids')
```

Reply:
I changed the order of the variable declarations and renamed `output_path` to `bids_path`. Hopefully it's clearer now.